Assets/LeapMotion/Scripts/Hands/HandTransitionBehavior.cs

```diff
@@ -4,7 +4,17 @@
 namespace Leap {
   public abstract class HandTransitionBehavior : MonoBehaviour {
 
-    public abstract void Reset();
-    public abstract void HandFinish();
+    protected abstract void HandReset();
+    protected abstract void HandFinish();
+    protected virtual void Awake(){
+      IHandModel iHandModel = GetComponent<IHandModel>();
```

Comment:
We might want to throw some null checks in here, that way we can throw warnings about HandTransitionBehavior requiring an IHandModel to function correctly, instead of throwing null reference exceptions if there isn't one.